source/server/http/admin.cc

```diff
@@ -279,11 +289,42 @@ Http::Code AdminImpl::handlerStats(const std::string&, Buffer::Instance& respons
     all_stats.emplace(gauge->name(), gauge->value());
   }
 
-  for (auto stat : all_stats) {
-    response.add(fmt::format("{}: {}\n", stat.first, stat.second));
+  if (params.size() == 0) {
+    // No Arguments so use the standard
+    for (auto stat : all_stats) {
+      response.add(fmt::format("{}: {}\n", stat.first, stat.second));
+    }
+  } else {
+    std::string format = params.begin()->first;
```

Comment:
Tried to use Field, FieldPtr and create json but not able to include them here.

Reply:
Yeah, thanks for trying. See TODO request above.